src/openms/source/DATASTRUCTURES/QTCluster.cpp

```diff
@@ -252,16 +257,31 @@ namespace OpenMS
     return quality_;
   }
 
+  double QTCluster::getCurrentQuality() const
+  {
+    // ensure cluster is finalized
+    OPENMS_PRECONDITION(finalized_,
+        "Cannot perform operation on cluster that is finalized")
+  
+    return quality_;
+  }
+
   void QTCluster::computeQuality_()
   {
     // ensure cluster is not finalized as we cannot call optimizeAnnotations_
     // in that case
     OPENMS_PRECONDITION(!finalized_,
         "Cannot perform operation on cluster that is finalized")
 
-    Size num_other = num_maps_ - 1;
+    // get references on member that is used in this function
+    NeighborMap& neighbors_ = data_->neighbors_;
```

Comment:
this variable is only used inside the `if` below?!
Then move it in there for better scoping.
You can even get rid of it by just
```
for (const auto& neighbor : data_->neighbors_)
{
  internal_distance += neighbor .second.first;
}
internal_distance += (num_other - data_->neighbors_.size()) * max_distance_;
```

Reply:
The variable is needed once more after the if. Ranged for loop done.